example/.aspect/cli/config.yaml

```diff
@@ -7,3 +7,4 @@ lint:
     - //tools/lint:linters.bzl%pmd
     - //tools/lint:linters.bzl%ruff
     - //tools/lint:linters.bzl%vale
+    - //tools/lint:linters.bzl%tfsec
```

Comment:
nit: consider adding to `lint.sh` as well so that it can be exercised in the "I hate Aspect CLI" use case